### PR TITLE
[docs] [integrations] akka persistence edits

### PR DESCRIPTION
**Pull request by aishwarya24**
- fixed links and modified wording of some steps based on @HarshDaryani896's feedback.
@netlify /preview/integrations/akka-ysql/

### CODE REVIEW

File: docs/content/preview/integrations/akka-ycql.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ The following example is inspired from the [akka-cassandra-demo](https://github.
 
     Press enter when you're prompted to name the application.
 
-1. Add the following code in `build.sbt` file from your project's home directory.
+1. Replace the existing code in `build.sbt` file from your project's home directory with the following:
```
Comment (ddhodge) [inline docs/content/preview/integrations/akka-ycql.md:58]:
```suggestion
1. Replace the existing code in the `build.sbt` file from your project's home directory with the following:
```

File: docs/content/preview/integrations/akka-ysql.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ menu:
   preview_integrations:
     identifier: ysql-akka
     parent: integrations
-    weight: 572
+    weight: 571
 type: docs
 ---
```
Comment (ddhodge) [inline docs/content/preview/integrations/akka-ysql.md:12]: quick start link redirects

File: docs/content/preview/integrations/akka-ycql.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ menu:
   preview_integrations:
     identifier: ycql-akka
     parent: integrations
-    weight: 571
+    weight: 572
 type: docs
 ---
```
Comment (ddhodge) [inline docs/content/preview/integrations/akka-ycql.md:12]: quick start link redirects

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/integrations/akka-ysql.md [hunk 4]
```diff
@@ -263,13 +263,13 @@ To write a sample application and customize its configuration, do the following:
     ```conf
```
Comment (HarshDaryani896) [inline docs/content/preview/integrations/akka-ysql.md:263]: small suggestion: mkdir resources -> mkdir src/main/resources

**Review by HarshDaryani896 — APPROVED** [review]
LGTM. Thank you @aishwarya24.

**Pull request merged**